cron/healthAndStatus.py

```diff
@@ -156,6 +167,11 @@ def initConfig():
     options.esservers = list(getConfig('esservers',
                                        'http://localhost:9200',
                                        options.configfile).split(','))
+    # configure the index to save events to
+    options.index = getConfig('index', 'mozdefstate', options.configfile)
+    # point to mapping json for the index
+    default_mapping_location = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'mozdefStateMappingTemplate.json')
```

Comment:
Found a bug here, this filename doesn't match the one committed, will push an update for this after lunch

Reply:
Seems I committed 2 files for mapping, going to remove one in favor of the other. 
mozdefStateMappingTemplate.json will be removed
mozdefStateDefaultMappingTemplate.json will be retained.